### PR TITLE
Iteration log updates

### PR DESCRIPTION
**Pull request by lguychard**

### CODE REVIEW

File: handbook/engineering/search/iteration_log.md [hunk 1]
```diff
@@ -23,6 +23,34 @@ This document contains the goals and work log for the search team's [2-week iter
 - **Work log:**
     - YYYY-MM-DD: $UPDATE
 
+### Implement `contains` filter predicate
+
+- **Owner(s):** Rijnard, Camden
+- **Outcomes:**
+    - $DESIRED_OUTCOME
+- **Work log:**
+    - YYYY-MM-DD: $UPDATE
```
Comment (lguychard) [inline handbook/engineering/search/iteration_log.md:32]: @rvantonder @camdencheek can you edit this to reflect your goals for this iteration (and add any missing sections if appropriate)

File: handbook/engineering/search/iteration_log.md [hunk 2]
```diff
@@ -23,6 +23,34 @@ This document contains the goals and work log for the search team's [2-week iter
...
+
+### Code Monitoring
+- **Owner(s):** Juliana
+- **Outcomes:**
+    - $DESIRED_OUTCOME
+- **Work log:**
+    - YYYY-MM-DD: $UPDATE
```
Comment (lguychard) [inline handbook/engineering/search/iteration_log.md:39]: @limitedmage can you edit this to reflect your goals for this iteration (and add any missing sections if appropriate)? Thanks!

File: handbook/engineering/search/iteration_log.md [hunk 3]
```diff
@@ -23,6 +23,34 @@ This document contains the goals and work log for the search team's [2-week iter
...
+
+### Exhaustive
+- **Owner(s):** $OWNER
+- **Outcomes:**
+    - $DESIRED_OUTCOME
+- **Work log:**
+    - YYYY-MM-DD: $UPDATE
```
Comment (lguychard) [inline handbook/engineering/search/iteration_log.md:46]: @keegancsmith @stefanhengl can you edit this to reflect your goals for this iteration (and add any missing sections if appropriate)? Thanks!

File: handbook/engineering/search/iteration_log.md (hunk 1 above)
Comment (rvantonder) [reply handbook/engineering/search/iteration_log.md]: Added in `main` and rebased this PR so we don't have to sync on everybody adding before merging this PR. But for some reason GH thinks that the comment to Juliana is now outdated, so maybe that wasn't ideal.

**Pull request merged**